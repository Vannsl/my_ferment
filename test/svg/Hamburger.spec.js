import { shallowMount } from '@vue/test-utils'
import HamburgerSvg from '@/components/svg/Hamburger'

const factory = () => {
  return shallowMount(HamburgerSvg)
}

describe('HamburgerSvg', () => {
  it('is a Vue instance', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
