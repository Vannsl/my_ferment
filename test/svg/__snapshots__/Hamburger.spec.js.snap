// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HamburgerSvg renders properly 1`] = `"<svg viewBox=\\"0 0 20 20\\" xmlns=\\"http://www.w3.org/2000/svg\\" class=\\"fill-current h-3 w-3\\"><title>Menu</title> <path d=\\"M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z\\"></path></svg>"`;
